x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/registered_domain.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { FieldNameField } from './common_fields/field_name_field';
+import { TargetField } from './common_fields/target_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+
+export const RegisteredDomain: FunctionComponent = () => {
+  return (
+    <>
+      <FieldNameField
+        helpText={i18n.translate(
+          'xpack.ingestPipelines.pipelineEditor.registeredDomain.fieldNameHelpText',
+          { defaultMessage: 'Field containing the source FQDN.' }
+        )}
+      />
+
+      <TargetField />
+
+      <IgnoreMissingField />
```

Comment:
Per the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/registered-domain-processor.html), the `ignore_missing` parameter is `true` by default. It looks like this component sets it to `false` by default.

Reply:
Boop, totally missed that! Ill patch it csv processor in another pr